evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -363,12 +362,11 @@ def make_dataframe(self, index, y_pred, y_true, scores, dataframe_index):
 
 
 class _SHAPTable(_SectionMaker):
-    def __init__(self, top_k_features, include_shap_values, training_data):
+    def __init__(self, top_k_features, include_shap_values):
         self.top_k_features = top_k_features
         self.include_shap_values = include_shap_values
-        self.training_data = training_data
 
-    def make_text(self, index, pipeline, input_features, y):
+    def make_text(self, index, pipeline, pipeline_features, y):
```

Comment:
I think this is mostly internal but as we change this up, could be helpful to add docstring for each arg? c:

Reply:
Sure!